fishtank/src/cluster.ts

```diff
@@ -15,12 +15,23 @@ export class Cluster {
     this.backend = options.backend ?? new Docker()
   }
 
+  private networkName(): string {
```

Comment:
nit, is it necessary to have this private function, you can just use `this.name` instead of `this.networkName()`? Let me know if I miss something or you plan to add more logic in this function.

Reply:
This is just for consistency, and for the sake of being explicit: there's a `containerName()`, there's probably going to be a `volumeName()`, and so on. I do not plan to add any logic to it.

Reply:
If we have multiple network, this will have additional logic right?! I think this function is for extensibility, currently the value is interchangeable, but it can extend easily to handle name logic.

Reply:
If we have multiple networks, yes, this might have a bit more logic (like `containerName()`). We do not have any plans to support multiple networks in our first iteration of the project though. No feature on the specification document calls for it.

Reply:
Yeah, totally not for the first milestone. For the future one like this https://www.notion.so/Specification-9eae768c51b849c490eea5faffcafbf0?pvs=4#1825ef0c63cf43f8813ab5e021ca88f5

Reply:
Network partitions can be simulated within a single network interface in many ways: by changing routing tables, through iptables, tc, BPF, or even by writing a custom gateway (though that's probably a bit excessive). I would argue some of the ways I mentioned are way easier to implement than having to deal with multiple Docker networks. Regardless, if we really want to support multiple Docker networks, there's nothing that prevents us from adding this functionality in the future.